crates/sui-adapter-transactional-tests/tests/programmable/large_vector@v2.snap

```diff
@@ -14,8 +14,8 @@ task 1, lines 8-14:
 //> 3: MakeMoveVec<vector<vector<vector<u64>>>>([Result(2), Result(2), Result(2), Result(2), Result(2), Result(2), Result(2), Result(2)]);
 //> 4: MakeMoveVec<vector<vector<vector<vector<u64>>>>>([Result(3), Result(3), Result(3), Result(3), Result(3), Result(3), Result(3), Result(3)]);
 //> 5: MakeMoveVec<vector<vector<vector<vector<vector<u64>>>>>>([Result(4), Result(4), Result(4), Result(4), Result(4), Result(4), Result(4), Result(4)]);
-Error: Transaction Effects Status: Move vector element (passed to MakeMoveVec) with size 1067301 is larger than the maximum size 1048576. Note that this maximum is scaled based on the type of the vector element.
-Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: MoveVectorElemTooBig { value_size: 1067301, max_scaled_size: 1048576 }, source: Some("Serialized bytes of value too large"), command: Some(5) } }
+mutated: object(0,1)
```

Comment:
We no longer have this hard element bound and are planning on relying on gas here, hence the very high computation cost. But we can adjust if we want